Casks/ihcask.rb

```diff
@@ -0,0 +1,11 @@
+cask "ihcask" do
```

Comment:
Is `ihcask` the right name for this? Maybe it should be `ih-rancher`, then we can add more casks with different names if needed.